lib/web_ui/lib/src/engine/semantics/link.dart

```diff
@@ -0,0 +1,105 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:ui/ui.dart' as ui;
+
+import '../dom.dart';
+import '../platform_dispatcher.dart';
+import 'semantics.dart';
+
+/// Manages semantics objects that represents a link.
+///
+/// This role is implemented via a anchor HTML element.
+class Link extends PrimaryRoleManager {
+  Link(SemanticsObject semanticsObject) : super.blank(PrimaryRole.link, semanticsObject) {
+    _setupDomElement();
+  }
+
+  /// The element used for link, i.e. `<a>`.
+  late DomHTMLElement anchorElement;
```

Comment:
There are still some more work needed to wire up the semantics action to this element, but I would like to know whether this is the way we want to pursuit.

An alternative is to Make semanticsObject.element to be a <a> directly, that way I can reuse all the RoleManager like Tappable or Focusable.